README.md

```diff
@@ -1,3 +1,8 @@
+<picture>
+  <source media="(prefers-color-scheme: dark)" srcset="./gui/public/StanPlay_Logo_RGB_White.png">
+  <img alt="The BridgeStan logo" src="./gui/public/StanPlay_Logo_RGB_Full.png" align="right" width=25%>
```

Comment:
BridgeStan --> Stan Playground (or StanPlayground or Stan-Playground, however we're styling it)